### PR TITLE
Update software plan

### LINKED ISSUE #45: typo for "adheres"
In https://docs.radiotherapy.ai/drafts/software-development-maintenance-plan.html
section 7 Verification Activities:
`All code ad-hears to the black style guide`
should be
All code adheres to the black style guide

### PR DESCRIPTION
**Pull request by SimonBiggs**
Fixes #45.

Updates details based on new local only software architecture.

### CODE REVIEW

File: docs/drafts/software-development-maintenance-plan.md
Comment (sjswerdloff) [inline docs/drafts/software-development-maintenance-plan.md:174]: You mentioned `pylance` earlier in addition to `pyright`
you don't mention `pylance` here

**Review by sjswerdloff — APPROVED** [review]

File: docs/drafts/software-development-maintenance-plan.md
Comment (SimonBiggs) [reply docs/drafts/software-development-maintenance-plan.md:174]: Ah yes. They're actually the same tool (sort of). Pylance is VSCode's utilisation of pyright with a few extra additions. When it is called from the command line one uses pyright directly (and can't use pylance in the CI suite). When in use within vscode one is using pyright via pylance.

https://github.com/microsoft/pylance-release/blob/main/FAQ.md

**Comment by SimonBiggs** [comment]
Sorry for the re-request review on this one. I have the setting set where any changes after an approval to a PR require re-approval.

The only change since your last approval was a commit to deal with a merge conflict:

https://github.com/RadiotherapyAI/RadiotherapyAI/pull/48/commits/a413de1bba9ee891792275c2758b5e20445683e3

In a follow up PR I'll make it so that file is allowed to be resolved "additively", make merge conflicts for that file less common.

**Review by sjswerdloff — APPROVED** [review]

**Pull request merged**
